source/core/owt_base/VideoFrameConstructor.cpp

```diff
@@ -311,22 +111,28 @@ bool VideoFrameConstructor::setBitrate(uint32_t kbps)
     return true;
 }
 
-std::vector<webrtc::SdpVideoFormat> VideoFrameConstructor::GetSupportedFormats() const {
-    return std::vector<webrtc::SdpVideoFormat>{
-        webrtc::SdpVideoFormat(
-            webrtc::CodecTypeToPayloadString(webrtc::VideoCodecType::kVideoCodecVP8)),
-        webrtc::SdpVideoFormat(
-            webrtc::CodecTypeToPayloadString(webrtc::VideoCodecType::kVideoCodecVP9)),
-        webrtc::SdpVideoFormat(
-            webrtc::CodecTypeToPayloadString(webrtc::VideoCodecType::kVideoCodecH264)),
-        webrtc::SdpVideoFormat(
-            webrtc::CodecTypeToPayloadString(webrtc::VideoCodecType::kVideoCodecH265))
-    };
+void VideoFrameConstructor::onAdapterFrame(const Frame& frame) {
+    deliverFrame(frame);
 }
 
-std::unique_ptr<webrtc::VideoDecoder> VideoFrameConstructor::CreateVideoDecoder(
-    const webrtc::SdpVideoFormat& format) {
-    return std::make_unique<AdaptorDecoder>(this);
+void VideoFrameConstructor::onAdapterStats(const AdapterStats& stats) {
+    if (m_videoInfoListener) {
+        std::ostringstream json_str;
+        json_str.str("");
+        json_str << "{\"video\": {\"parameters\": {\"resolution\": {"
+                    << "\"width\":" << stats.width << ", "
+                    << "\"height\":" << stats.height
+                    << "}}}}";
+        m_videoInfoListener->onVideoInfo(json_str.str().c_str());
+    }
+}
+
+void VideoFrameConstructor::onAdapterData(char* data, int len) {
```

Comment:
the feedback callback is defined w/o return for video, but with return for audio. Any reason?